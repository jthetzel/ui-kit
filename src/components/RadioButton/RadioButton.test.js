import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import RadioButton from '../RadioButton';

test('RadioButton does not crash', () => {
  const tree = shallow(<RadioButton value>title</RadioButton>);
  expect(toJson(tree)).toMatchSnapshot();
});
