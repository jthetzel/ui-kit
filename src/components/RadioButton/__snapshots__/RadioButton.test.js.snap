// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton does not crash 1`] = `
<div
  className="css-173444m e1dn8rdo0"
  value={true}
>
  title
</div>
`;
